analysis/analysis.py

```diff
@@ -24,11 +30,15 @@
 Returns:
     padded_sequences: the padded sequences to be analyzed by the model
 """
-def pre_process(tokenizer:Tokenizer, data, max_word_len):
+
```

Comment:
I'm not sure how I feel about 2 lines between the function's info and the function itself. I personally wouldn't go past 1 line. I think at the least, the lines between the previous code and the info should be greater than the lines between the info and the function if that makes sense. I'd like to hear what others' opinions are one it.

Reply:
I actually agree, but the code just kept re-formatting on save and I didn't feel like fixing it lol.

Reply:
Oh interesting. Maybe you have some additional plug-in doing that.
So should we go with one line between the info and the function? Or no extra line?
And then what about between the previous code and the info? If we go with no lines for the question above ^ , then I'm thinking just one line. If not, then two lines.